packages/caliper-samples/src/contract/scalardl/src/main/java/com/example/contract/smallbank/SendPayment.java

```diff
@@ -0,0 +1,47 @@
+package com.example.contract.smallbank;
+
+import com.scalar.ledger.asset.Asset;
+import com.scalar.ledger.contract.Contract;
+import com.scalar.ledger.exception.ContractContextException;
+import com.scalar.ledger.ledger.Ledger;
+import java.util.Optional;
+import javax.json.Json;
+import javax.json.JsonObject;
+import javax.json.JsonObjectBuilder;
+
+public class SendPayment extends Contract {
```

Comment:
What is the difference between this and `SimpleTransfer` ?

Reply:
@jnmt You probably missed it.

Reply:
Sorry. I forgot to mention this. Basically the things what these two contracts are doing is same. But it cannot be unified because the applications use the different name of argument. E.g. "account1" vs. "source_customer_id"